setup.py

```diff
@@ -49,25 +58,25 @@
 
 
 setup(
-    name="<PYPI_NAME>",
+    name="docs-build",
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
-    description="""<PYPI_NAME>: <SHORT_DESCRIPTION>""",
+    description="docs-build: Build Sphinx Documentation for ApeWorX plugins",
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/<REPO_NAME>",
+    url="https://github.com/ApeWorX/docs-build",
```

Comment:
```suggestion
    url="https://github.com/ApeWorX/ape-docs-build",
```

Reply:
Actually - `ape-docs-build` might not be the best name either :( because it makes it sound like a plugins (since it starts with `ape-`.

perhaps:  `ApeDocumentationBuilder` ?